src/App.tsx

```diff
@@ -1,10 +1,26 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    {/* Keep this input for tests */}
-    <input type="text" />
+export const App: React.FC = () => {
+  const [pressedKey, setPressedKey] = useState('');
 
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+  const keyupHandler = (event: KeyboardEvent) => setPressedKey(event.key);
+
+  useEffect(() => {
+    document.addEventListener('keyup', keyupHandler);
+
+    return () => document.removeEventListener('keyup', keyupHandler);
+  }, []);
+
+  return (
+    <div className="App">
+      {/* Keep this input for tests */}
+      <input type="text" />
+
+      {
+        pressedKey
+          ? (<p className="App__message">{`The last pressed key is [${pressedKey}]`}</p>)
+          : (<p className="App__message">Nothing was pressed yet</p>)
+      }
```

Comment:
make it inside one `<p>`
```suggestion
  <p className="App__message">
    {pressedKey
       ? {`The last pressed key is [${pressedKey}]`})
       : Nothing was pressed yet}
   </p>
```